src/components/page-components/ExploreProjectPage/ExploreProjectPage.tsx

```diff
@@ -0,0 +1,57 @@
+import { sendToBackground } from "@plasmohq/messaging"
+
+import "./ExploreProjectPage.css"
+
+import { useEffect, useState } from "react"
+
+import BackButton from "~components/BackButton/BackButton"
+import CuratorDetails from "~components/CuratorDetails/CuratorDetails"
+import Footer from "~components/Footer/Footer"
+import ProjectDetails from "~components/ProjectDetails/ProjectDetails"
+import type { ProjectData } from "~types/projectTypes"
+
+const ExploreProjectPage = () => {
+  const [project, setProject] = useState<ProjectData>()
+
+  useEffect(() => {
+    const fetchProject = async () => {
+      const response = await sendToBackground({
+        name: "fetchProjectDetailsById",
+        body: {
+          id: 1
+        }
+      })
+      console.log({ response })
```

Comment:
One I think we might want to make a convention here. I've been adhering to the following on this codebase:

```js
const response: ProjectResponse = await ....
```

Since you set up types to account for the `response` object wrapping I think this would be way better because it actually triggers a runtime check. When needed the `as` syntax works fine because we do know that's what's coming out, but this way we make use of the detailed types you set up and actually buttress up our unit tests with this extra bit of static testing.

What do you think?

Also, please remove the console.log on line 24, it's going to drive us mad when we start working on another feature and it keeps popping up